Episode-5/after/Blapp/app/controllers/PostDetailController.rb

```diff
@@ -38,4 +38,4 @@ def saveButton
     @saveButton ||= UIBarButtonItem.alloc.initWithBarButtonSystemItem(UIBarButtonSystemItemSave, target:self, action:'save')
   end
 
-end
+end
```

Comment:
This is an example of a "white space" change.  Try to filter those out.  They don't really add any value.

Reply:
Alright